Add a recommendation to use LTS JDK versions (#14067)
DataDog/documentation: content/en/tracing/profiler/enabling/java.md

## Requirements

The Datadog Profiler requires [JDK Flight Recorder][2]. The Datadog Profiler library is supported in OpenJDK 11+, Oracle JDK 11+, [OpenJDK 8 (version 8u262+)][3] and Azul Zulu 8+ (version 8u212+). It is not supported in OpenJ9 as it doesn't support the [JDK Flight Recorder][2].

> LTS (Long Term Support) JDK versions (8, 11 and 17) are recommended as the non-LTS versions might be missing stability and performance fixes related to Datadog Profiler library.

Reviewer: ```suggestion

Because non-LTS JDK versions may not contain stability and performance fixes related to the Datadog Profiler library, use versions 8, 11, and 17 of the Long Term Support JDK.
```
